sdk/iot/hub/readme.md

```diff
@@ -0,0 +1,53 @@
+# Azure IoT Clients for Embedded C: Azure IoT Hub Client
+
+Official Embedded C client for Azure IoT Hub.
+
+## Getting started
+
+- **Docs**: If you're new to the Azure SDK for Embedded C take a look at the [State Machine diagram](https://github.com/Azure/azure-sdk-for-c/blob/master/sdk/iot/doc/mqtt_state_machine.md#device-provisioning-service) that explains the high-level architecture, SDK components, and a clear view of SDK x Application responsibilities. 
+- **Samples**: [This page](https://github.com/Azure/azure-sdk-for-c/tree/master/sdk/iot/hub/samples) explains samples for the Azure SDK for Embedded C and how to use them.
+
+[TODO] update wiht the list of samples Hub, CDD, telemetry, method and twin
```

Comment:
wiht?

Reply:
Yea I think I had the same comment on wellington's PR. Its probably best to merge his first and then this one.

Reply:
Removed this TODO line from the page